oaknational/Oak-Web-Application
# feat: blog-categories

## Description

- currently uses blog categories from all blogs query (rather than a separate query for getting blog categories)
- tablet view uses desktop view
- current url structure (though i feel we should discuss it a little before going live with this) is:
  - "all blogs" is at `/blog`
  - blog index by category is at `/blog/category/{category-slug}
  - blog post is at `/blog/{blog-post-slug}`
  - what i was unsure about was whether category should be part of search params rather than in the path 

**Need to add some tests and add mobile blog filters storybook story**

## Issue(s)

Fixes #692 
Progresses #434

## How to test

1. Go to {cloud link}
2. See home screen blog list should have links to categories
3. Clicking on one should take you to /blog/category/{category slug} 
4. You should see on the category list, with the current category selected
5. Clicking on a category in that list should take you to that category blog index page

## Screenshots

![image](https://user-images.githubusercontent.com/12934669/191691345-f0163c53-9319-4b38-8a0d-6f7023846934.png)

![image](https://user-images.githubusercontent.com/12934669/191691417-be2c5bad-4e1e-4401-9501-ab8ab8b01b0d.png)

![image](https://user-images.githubusercontent.com/12934669/191691447-b1b73a7b-b5db-4c64-8f90-21730f7e19ca.png)

![image](https://user-images.githubusercontent.com/12934669/191691531-448926f6-7a0b-4aff-8f81-029a0aeff312.png)

![image](https://user-images.githubusercontent.com/12934669/191692687-b1f7a52b-6717-413d-88ba-074b6af2c0c3.png)


## Checklist
